schedule/yast/textmode_installation_minimal_role/textmode_installation_minimal_role_x86_64.yaml

```diff
@@ -0,0 +1,27 @@
+---
+name:           textmode_installation_minimal_role
+description:    >
+  Full Medium installation that covers the following cases:
+    1. Installation in textmode;
+    2. "Minimal" role is selected;
+    3. Boot to command-line mode;
+    4. Installation is validated by default set of smoke tests.
+vars:
+  VIDEOMODE: text
+  SYSTEM_ROLE: minimal
+  DESKTOP: textmode
+  YUI_REST_API: 1
+schedule:
+  system_role:
+    - installation/system_role/select_role_minimal
+  system_preparation:
```

Comment:
let's validate default target as well.

Reply:
added

Reply:
missing test module to validate as well.

Reply:
changed.